examples/dockerfile/BUILD.bazel

```diff
@@ -87,3 +89,85 @@ assert_oci_image_command(
            ||     ||
 """,
 )
+
+# This tar can be arranged in arbitrary ways.
+# For example, one could grab pkg_files from elsewhere in the workspace.
+# Anything in the tar can be used by a Dockerfile ADD or COPY.
+pkg_tar(
```

Comment:
actually can we not use pkg_tar here? we have bazel-lib dependency here already. you can use https://github.com/bazel-contrib/bazel-lib/blob/main/docs/tar.md

(i don't want a pkg_tar dependency even if it's dev_dep)

Reply:
I might need some assistance with this one.

When I try `bazel build //examples/dockerfile:buildx_context` I get:

```
tar: Unrecognized archive format
tar: Error exit delayed from previous errors.
Target //examples/dockerfile:buildx_context failed to build
```

Any ideas?

Reply:
See my comment above.